src/jdk.javadoc/share/classes/jdk/javadoc/doclet/package-info.java

```diff
@@ -51,9 +51,9 @@
  * The invocation is defined by the interface {@link jdk.javadoc.doclet.Doclet}
  * -- the {@link jdk.javadoc.doclet.Doclet#run(DocletEnvironment) run} interface
  * method, defines the entry point.
- * <pre>
- *    public boolean <b>run</b>(DocletEnvironment environment)
- * </pre>
+ * {@snippet id="entry-point" type=java :
```

Comment:
What does the `type` attribute do? It's not mentioned in the JEP.

Reply:
Should probably be `lang=java`.